### ISSUE TITLE
MissingPluginException(No implementation found for method dispose on channel com.ryanheise.just_audio.methods)

### ISSUE THREAD

**Issue by osukono** [ryanheise/just_audio#452]
Starting from 0.9.0, version, I get an error when calling `AudioPlayer.stop()` method.

```
I/ExoPlayerImpl(21048): Init ffe29aa [ExoPlayerLib/2.13.1] [generic_x86, Android SDK built for x86, Google, 29]
E/ExoPlayerImplInternal(21048): Playback error
E/ExoPlayerImplInternal(21048):   com.google.android.exoplayer2.ExoPlaybackException: Source error
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java:579)
E/ExoPlayerImplInternal(21048):       at android.os.Handler.dispatchMessage(Handler.java:103)
E/ExoPlayerImplInternal(21048):       at android.os.Looper.loop(Looper.java:214)
E/ExoPlayerImplInternal(21048):       at android.os.HandlerThread.run(HandlerThread.java:67)
E/ExoPlayerImplInternal(21048):   Caused by: com.google.android.exoplayer2.upstream.HttpDataSource$HttpDataSourceException: Unable to connect
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:356)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.DefaultDataSource.open(DefaultDataSource.java:201)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.StatsDataSource.open(StatsDataSource.java:84)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.source.ProgressiveMediaPeriod$ExtractingLoadable.load(ProgressiveMediaPeriod.java:1015)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.Loader$LoadTask.run(Loader.java:415)
E/ExoPlayerImplInternal(21048):       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/ExoPlayerImplInternal(21048):       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/ExoPlayerImplInternal(21048):       at java.lang.Thread.run(Thread.java:919)
E/ExoPlayerImplInternal(21048):   Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(ConscryptFileDescriptorSocket.java:231)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.io.RealConnection.connectTls(RealConnection.java:196)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.io.RealConnection.connectSocket(RealConnection.java:153)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.io.RealConnection.connect(RealConnection.java:116)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.http.StreamAllocation.findConnection(StreamAllocation.java:186)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection(StreamAllocation.java:128)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.http.StreamAllocation.newStream(StreamAllocation.java:97)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:289)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:232)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:465)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect(HttpURLConnectionImpl.java:131)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.connect(DelegatingHttpsURLConnection.java:90)
E/ExoPlayerImplInternal(21048):       at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:30)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.makeConnection(DefaultHttpDataSource.java:641)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.makeConnection(DefaultHttpDataSource.java:543)
E/ExoPlayerImplInternal(21048):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:349)
E/ExoPlayerImplInternal(21048):       ... 7 more
E/ExoPlayerImplInternal(21048):   Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.TrustManagerImpl.verifyChain(TrustManagerImpl.java:674)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.TrustManagerImpl.checkTrustedRecursive(TrustManagerImpl.java:551)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:507)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:426)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.TrustManagerImpl.getTrustedChainForServer(TrustManagerImpl.java:354)
E/ExoPlayerImplInternal(21048):       at android.security.net.config.NetworkSecurityTrustManager.checkServerTrusted(NetworkSecurityTrustManager.java:94)
E/ExoPlayerImplInternal(21048):       at android.security.net.config.RootTrustManager.checkServerTrusted(RootTrustManager.java:89)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.Platform.checkServerTrusted(Platform.java:224)
E/ExoPlayerImplInternal(21048):       at com.android.org.conscrypt.ConscryptFileDescriptorSocket.verifyCertificateChain(ConscryptFileDescriptorSo
E/AudioPlayer(21048): TYPE_SOURCE: Unable to connect
I/ExoPlayerImpl(21048): Release ffe29aa [ExoPlayerLib/2.13.1] [generic_x86, Android SDK built for x86, Google, 29] [goog.exo.core]
E/MethodChannel#com.ryanheise.just_audio.methods( 5649): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/MethodChannel#com.ryanheise.just_audio.methods( 5649): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

```
I/ExoPlayerImpl( 5649): Init 13d4ca8 [ExoPlayerLib/2.13.1] [generic_x86_arm, AOSP on IA Emulator, Google, 28]
E/flutter ( 5649): [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: MissingPluginException(No implementation found for method dispose on channel com.ryanheise.just_audio.methods.d0834e4f-c449-491e-83d3-1b8b3e11fa0b)
E/flutter ( 5649): #0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #1      MethodChannelAudioPlayer.dispose
package:just_audio_platform_interface/method_channel_just_audio.dart:158
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #2      AudioPlayer._disposePlatform
package:just_audio/just_audio.dart:1299
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #3      AudioPlayer._setPlatformActive.setPlatform
package:just_audio/just_audio.dart:1126
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649):
E/flutter ( 5649): [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: MissingPluginException(No implementation found for method dispose on channel com.ryanheise.just_audio.methods.d0834e4f-c449-491e-83d3-1b8b3e11fa0b)
E/flutter ( 5649): #0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #1      MethodChannelAudioPlayer.dispose
package:just_audio_platform_interface/method_channel_just_audio.dart:158
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #2      AudioPlayer._disposePlatform
package:just_audio/just_audio.dart:1299
E/flutter ( 5649): <asynchronous suspension>
E/flutter ( 5649): #3      AudioPlayer._setPlatformActive.setPlatform
package:just_audio/just_audio.dart:1126
E/flutter ( 5649): <asynchronous suspension>

```

**Comment by ryanheise** [comment]
There was a `stop` bug introduced over the weekend which I've fixed in 0.9.2 but it was a NPE bug and didn't have exactly the same error message as the one above. If you'd like me to investigate the above bug, I'll need a valid bug report as per the instructions since it provides me with all of the information I need to reproduce and investigate.

**Comment by osukono** [comment]
Thank you very much. 0.9.2 fixed my problem.
